index.html

```diff
@@ -12,6 +12,57 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header class="navbar-container">
+        <img class="hamburger" src="/home/dom/Documents/HTML-CSS-Coursework-Week2/img/menu-hamburger.svg" width="3%" height="3%" alt="logo"><!--little logo missing-->
+        <img class="logo" src="/home/dom/Documents/HTML-CSS-Coursework-Week2/img/karma-logo.svg" width="3%" height="3%" alt="logo"><!--little logo missing-->
+        <nav>
+            <ul>
+                <li><a class="navLink" href="">Meet Karma</a></li>
+                <li><a class="navLink" href="">How it Works</a></li>
+                <li><a class="navLink" href="">Store</a></li>
+                <li><a class="navLink" href="">Blog</a></li>
+                <li><a class="navLink" href="">Help</a></li>
+                <li><a class="navLink" href="">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+        <div class="container">
+            <h2>Introducing Karma</h2>
+            <h3>Bring WiFi with you, everywhere you go.</h3>
+            <button class="cta">Learn More</button>
+        </div>
+        <section>
+            <h1>Everyone needs a little Karma.</h1>
```

Comment:
This is the heading for the next section, so you don't need a separate `<section>` tag here. You can move this heading directly inside the next section, before the `ul`